dependencies/build_darktable.sh

```
@@ -0,0 +1,100 @@
+  --enable-imagemagick \
+  --enable-jxl \
+  --enable-opencl \
+  --enable-openexr \
+  --enable-openmp \
+  --enable-webp"
+
+./build.sh \
+  --prefix "/opt/darktable" \
+  --build-type "Release" \
+  --install \
+  ${FEATURES}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)